work_items/kyc-vcs/requirements-to-accept-a-kyc-vc.md

```diff
@@ -0,0 +1,576 @@
+
+```json
+{
+   "@context": ["https://www.w3.org/2018/credentials/v1"],
+   "id": "8beaf6db-c636-4874-9997-445b31ea2596",
+   "type": ["VerifiableCredential", "EU-UK-KYC"],
+   "issuer": "did:example:KYC-Issuer",
+   "credentialSubject": {
+      "id": "did:example:1a2b3c4d5e6f7g",
+      "name": {
+         "familyName": "Biche",
+         "givenName": "Jean"
+      },
+      "birthDate": "1999/12/25",
+      "postalAddress": {
+         "streetAddress": "31 rue Perdu",
+         "country": "FRA",
+         "locality": "La Ville",
+         "region": "Ile-de-France",
+         "postalCode": "75000"
+      },
+      "nationality": "FRA",
+      "accountType": "DepositAccount",
+      "purposeOfAccount": "Deposits",
+      "occupationalCategory": "3123 Construction Supervisors",
+      "sourceOfFunds": "Employment" 
+   },
+   "proof": { "..." }
+}
+```
+
+**Note:** EU and UK AML regulations require both a Photo ID and address verification, so to meet compliance requirements in these jurisdictions the verifiable credential will require the additional evidence outlined in Section 3 and 4 below to be considered valid.
+
+
+### 3. Identity Verification Evidence Vocabulary
+
+In order for the KYC Verifiable Credential to be accepted by most financial institutions, it needs to be verified using a standard method that is recognized by others. The following table outlines the information that should be included in order to provide evidence of identity verification (regardless of whether the documentary or non-documentary method is used). It is noted that the verification requirements differ by country and are more prescriptive in certain countries. Some countries may also require more than one method of verification (such as in Canada).
+
+
+<table>
+  <tr>
+   <td><strong>Vocabulary</strong>
+   </td>
+   <td><strong>Field Description & References</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Method:
+   </td>
+   <td>Documentary, non-documentary
+   </td>
+  </tr>
+  <tr>
+   <td>Source:
+   </td>
+   <td>Photo ID, public records, official document, other document (<em>we need to determine the full list of acceptable documents</em>), note that this field is a high level description of the source, the document type is included below
+   </td>
+  </tr>
+  <tr>
+   <td>Match Result:
+   </td>
+   <td>Full match (all data elements matched), partial match (3 out of 4 elements matched), match with issues (2 out of 4 elements matched), unmatched <em>(the issuer should not issue a VC based on an unmatched result)</em>
+   </td>
+  </tr>
+  <tr>
+   <td>Unmatched Data:
+   </td>
+   <td>Name, date of birth, address, TaxID or ID Number <em>(could be one or many)</em>. It is important to provide information on unmatched data, this does not make the VC invalid, but rather demonstrates the quality of the verification process
+   </td>
+  </tr>
+  <tr>
+   <td>Verifier Name:
+   </td>
+   <td>Free text (examples Jumio, Onfido, Prove, etc.)
```

Comment:
```suggestion
   <td>Free text (examples Jumio, Onfido, Prove, etc.) suggest using the name here instead of a DID so that the Verifier does not need to resolve the DID information
```